jenkinsfiles/ginkgo-runtime-kernel.Jenkinsfile

```
@@ -14,9 +14,9 @@ pipeline {
             returnStdout: true,
             script: 'echo -n "${JobKernelVersion}"'
             )}"""
-        NETNEXT="""${sh(
+        KERNEL="""${sh(
             returnStdout: true,
-            script: 'if [ "${JobKernelVersion}" = "net-next" ]; then echo -n "1"; else echo -n "0"; fi'
+            script: 'if [ "${JobKernelVersion}" = "net-next" ]; then echo -n "netnext"; else echo -n ""; fi'
```

I don't know if this should be updated as well? I'm not sure where this variable is set and used.
```suggestion
            script: 'if [ "${JobKernelVersion}" = "netnext" ]; then echo -n "netnext"; else echo -n ""; fi'
```